spec/handlers/sessions_create_spec.rb

```diff
@@ -1,199 +1,96 @@
 # coding: utf-8
 require 'rails_helper'
+require_relative 'sessions_create_shared_examples'
```

Comment:
Why did all this code move to shared examples if only used here?

Reply:
It's used in here in 2 places: https://github.com/openstax/accounts/pull/594/files/2be24f2eb634048763f81397d754701f890a0b67#diff-c80901fc9ff3f8b10d4b0564dd4e2b85R7 and https://github.com/openstax/accounts/pull/594/files/2be24f2eb634048763f81397d754701f890a0b67#diff-c80901fc9ff3f8b10d4b0564dd4e2b85R27

Idk do you want me to move the shared examples to the top/bottom of this file?